sidecar/sokuji_sidecar/tts_voices.py

```
@@ -8,6 +8,33 @@
 
 
+def _repo_cached(repo: str) -> bool:
+    """True if `repo`'s snapshot is present in the local HF cache (best-effort:
+    unlike native_models._repos_cached, doesn't check for a stray .incomplete
+    blob — used only to pick which cached variant repo to read voices/ from,
+    not as a download-readiness gate)."""
+    from huggingface_hub import snapshot_download
+    try:
+        snapshot_download(repo_id=repo, local_files_only=True)
+        return True
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)